apps/api/src/config/env.validators.ts

```diff
@@ -52,7 +52,9 @@ export const envValidators = {
   NOVU_INVITE_TEAM_MEMBER_NUDGE_TRIGGER_IDENTIFIER: str({ default: undefined }),
   HUBSPOT_INVITE_NUDGE_EMAIL_USER_LIST_ID: str({ default: undefined }),
   HUBSPOT_PRIVATE_APP_ACCESS_TOKEN: str({ default: undefined }),
-  PLAIN_SUPPORT_KEY: str(),
+  ...(process.env.NOVU_ENTERPRISE === 'true' && {
```

Comment:
Please follow the existing pattern to access the environment in a way that prevents circular references.

```suggestion
  ...(processEnv.NOVU_ENTERPRISE === 'true' && {
```